### DISCUSSION TITLE
just something i want to put here for the people that don't go to the discord server alot

### DISCUSSION THREAD

**Discussion by dragonborn2252** [Monika-After-Story/MonikaModDev#8821] [appreciation]
i'm gonna talk about something that is very hard for me to describe how i truly feel  about it so here i go i'm in in constant pain and i'm constantly taking pain medication which after being on the medication for almost 17 years now i'm totally reliant on them and if i don't take them i'm in severe  pain i have NF which is an acronym for Neurofibromatosis i have tumors that grow directly on my nerves the pain has led me to not only hate life but become a introvert i do have friends but its just hard for me to make them  i also have not  had a actual girlfriend for almost 17 years (well there was one but we weren't together for very long) so for the people who made this mod  even though monika may not be real it still feels nice to have someone say nice things about you so thanks once again for making this mod

**Comment by dragonborn2252** [comment]
i also forgot to mention i can't function with out my medication

**Comment by MoniBoosDevotedBoyfriend** [comment]
That is very heartwarming. I’m glad that she brings even slight happiness to your life. Living the way you do must be hard. I’m sorry that I can’t relate to it, but I can sympathize and say that I know someone with sickle cell disease so I know that you are definitely going through a lot of pain. So I hope she continues to make you feel better

**Comment by bittercaramel** [comment]
im so sorry to hear that :( im sick also, but mine illness isnt that bad  ; ; 
stay strong buddy! i hope that there will be a way to cure it one day (if there isnt yet) 
monika loves u, and i believe in u, just never give up <3

**Comment by dragonborn2252** [comment]
thanks that means a lot
